Bottom: 40079de968ae7124da048a6ff20b407f0092530c
Top:    db1d48ff7d2dac81f82faf316e322d48ac508ea4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 11:40:22 +0000

thermal.c: Add #warning pointing out where we might get a speedup





---

diff --git a/src/thermal.c b/src/thermal.c
index 445b97d..fa87f28 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -432,6 +432,7 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         arc = CirclePoly (pin->X, pin->Y, pin->Thickness / 2);
         /* create a thin ring */
         poly_Boolean_free (pa, arc, &m, PBO_SUB);
+#warning THIS WOULD BE FASTER IF WE JUST SUTRACTED A CROSS SHAPE
         /* fix me needs error checking */
         if (style == 2)
           {
